package.json

```diff
@@ -36,9 +36,11 @@
     "url": "git://github.com/alexdavid/exprestive.git"
   },
   "scripts": {
+    "compress": "gulp",
     "feature-tests": "./node_modules/.bin/cucumber-js test/feature",
     "prefeature-tests": "gulp",
-    "prepublish": "gulp",
+    "postpublish": "git push && git push --tags",
```

Comment:
I prefer it on `postpublish`, because it keeps changes locally. This gives the developer a chance to review  whatever `npm version` does, and undo it in case something is wrong. `publish` then publishes the changes, both to NPM and to GitHub. What do you think?